libs/atomic-elements/src/lib/components/GridSection/ContextMenu.tsx

```diff
@@ -93,13 +93,15 @@ const ContextMenu = (props: ContextMenuProps) => {
           open={Boolean(anchorEl)}
           onClose={handleClose}
         >
-          {props.attr.detailPage && (
-            <MenuItem onClick={handleDetailsClick(props.attr, props.cell)}>
-              Go to {props.attr.detailPage} details
-            </MenuItem>
-          )}
+          {props.attr.crossLinking &&
+            props.attr.crossLinking.length > 0 &&
```

Comment:
Instead of this you can just verify if is an Array, and `.map` is gonna generate nothing if is empty
```suggestion
          {Array.is(props.attr.crossLinking) &&
```